packages/@o3r/testing/src/core/playwright/element.ts

```diff
@@ -1,4 +1,4 @@
-import {Locator, Page} from 'playwright';
+import type {Locator, Page} from 'playwright';
 import {ElementProfile} from '../element';
```

Comment:
As we export a new type using these 2 types from playwright package ([few lines below](https://github.com/AmadeusITGroup/otter/blob/2c573973f167588597da5a4dc3710e22a9dfef91/packages/%40o3r/testing/src/core/playwright/element.ts#LL13C1-L16C3)) I am not sure we can keep `playwright` only as a devDep.
cc: @kpanot

Reply:
Indeed, it would need to be peerDep as well